app/controllers/foreman_wreckingball/hosts_controller.rb

```diff
@@ -112,6 +134,10 @@ def find_status
       @status = HostStatus::Status.find_by!(:id => params[:status_id], :host_id => @host.id)
     end
 
+    def find_statuses
+      @statuses = HostStatus::Status.includes(:host).find(params[:status_ids])
```

Comment:
Does this properly check permissions?

Reply:
checking permissions is not implemented yet, I'll add it with proper test